packages/dataviews/src/components/dataform/stories/index.story.tsx

```diff
@@ -174,6 +181,72 @@ const fields: Field< SamplePost >[] = [
 		type: 'text',
 		Edit: 'textarea',
 	},
+	{
+		id: 'comment_status',
+		label: 'Comment Status',
+		type: 'text',
+		Edit: 'radio',
+		elements: [
+			{ value: 'open', label: 'Allow comments' },
+			{ value: 'closed', label: 'Comments closed' },
+		],
+	},
+	{
+		id: 'ping_status',
+		label: 'Allow Pings/Trackbacks',
+		type: 'boolean',
+		Edit: 'checkbox',
```

Comment:
isn't `checkbox` the [default](https://github.com/WordPress/gutenberg/blob/4f8878dc60088933041e0eb8f95051a3cc1f0f33/packages/dataviews/src/field-types/boolean.tsx#L51)?

```suggestion
```